### PR TITLE
[docs] GettingStarted: Add a warning about spaces in the absolute path

### PR DESCRIPTION
**Pull request by simanerush**
Add a warning about spaces in the absolute path to the Getting Started document

### CODE REVIEW

File: docs/HowToGuides/GettingStarted.md [hunk 1]
```diff
@@ -66,6 +66,11 @@ toolchain as a one-off, there are a couple of differences:
    mkdir swift-project
    cd swift-project
    ```
+   
+> **Warning**  
+> Make sure the absolute path to your `swift-project` folder **does not** contain spaces, 
+    since that might cause build issues on the next steps.
```
Comment (AnthonyLatsis) [inline docs/HowToGuides/GettingStarted.md:72]: Does this sound better?

```suggestion
    since that might cause issues during the build step.
```

Comment (AnthonyLatsis) [inline docs/HowToGuides/GettingStarted.md:72]: The warning needs to be indented (see how it renders on GitHub).

File: docs/HowToGuides/GettingStarted.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ toolchain as a one-off, there are a couple of differences:
    mkdir swift-project
    cd swift-project
    ```
+   
+> **Warning**  
+> Make sure the absolute path to your `swift-project` folder **does not** contain spaces,
```
Comment (AnthonyLatsis) [inline docs/HowToGuides/GettingStarted.md:71]: Could we stick to "directory" for consistency?

**Review by AnthonyLatsis — APPROVED** [review]
A ridiculously easy thing to miss for a newcomer, and the issue would reveal itself through misleading CMake errors only halfway into building the project. Looks good, thank you!

File: docs/HowToGuides/GettingStarted.md [hunk 3]
```diff
@@ -66,6 +66,11 @@ toolchain as a one-off, there are a couple of differences:
    mkdir swift-project
    cd swift-project
    ```
+   
+    > **Warning**  
+    > Make sure the absolute path to your `swift-project` directory **does not** contain spaces, 
+        since that might cause build issues during the build step.
```
Comment (AnthonyLatsis) [inline docs/HowToGuides/GettingStarted.md:72]:
```suggestion
        since that might cause issues during the build step.
```

**Review by LucianoPAlmeida — APPROVED** [review]

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Comment by LucianoPAlmeida** [comment]
@swift-ci Please test Windows Platform

**Comment by xedin** [comment]
@swift-ci Please test Windows Platform

**Review by xedin — APPROVED** [review]

**Pull request merged**